buildbot/compile.py

```diff
@@ -5,20 +5,21 @@
 
 DEFAULT_CPU_COUNT = 4
 
-def do_compile(args):
-    ret = False
 
+def do_compile(args):
     cpu_count = os.cpu_count()
     if cpu_count is None:
         cpu_count = DEFAULT_CPU_COUNT
 
-    make_cmd = ["ninja", "-j", str(cpu_count), "deploy-sycl-toolchain"]
-    print(make_cmd)
+    commands = [["ninja", "-j", str(cpu_count), "deploy-sycl-toolchain"],
```

Comment:
I suggest keep using one command:
```suggestion
    make_cmd = ["ninja", "-j", str(cpu_count), "deploy-sycl-toolchain", "opencl-aot"]
```